website/content/gateway/api_reference/resources/credits/credits.md

```diff
@@ -32,6 +32,12 @@ POST https://gateway.clearhaus.com/credits
 {{% regex_optional %}}Optional{{% /regex_optional %}}
 {{% /description_details %}}
 
+{{% description_term %}}purpose_of_payment {{% regex %}}[A-Z]{6}{{% /regex %}}{{% /description_term %}}
+{{% description_details %}} The fields code shall reflect the nature and purpose of the transaction.
+Valid codes are list in Visas Purpose of Payment - Standardized Codes List.
+{{% regex_optional %}} Required for outbound transactions sent into Argentina, Bangladesh, Chile, Columbia, Egypt, India and Mexico. {{% /regex_optional %}}
```

Comment:
```suggestion
{{% regex_optional %}} Required for cross-border transactions destined for Argentina, Bangladesh, Chile, Columbia, Egypt, India and Mexico. {{% /regex_optional %}}
```

I would probably have gone with `cross-border` instead and think `into` is somewhat off :shrug:

Reply:
I updated the text here.
https://github.com/clearhaus/gateway-api-docs/pull/188/commits/2593828f3a0a4205009fddcd389921740045b7a1